js/client-battle-tooltips.js

```diff
@@ -35,13 +35,92 @@ var BattleTooltips = (function () {
 	BattleTooltips.hideTooltip = function () {
 		$('#tooltipwrapper').html('');
 	};
-
 	// tooltips
+	// Touch delay, pressing finger more than that time will cause the tooltip to open.
+	// Shorter time will cause the button to click
+	var touchDelayForTap = 300; // ms
+
+	// Each time a finger presses the button this function will be callled
+	// First finger starts the counter, and when last fingers leaves time is checked
+	BattleTooltips._handleTouchStartFor = function (event, roomid, thing, type, elem, ownHeight) {
+		// Prevent default on touch events to block mouse events when touch is used
+		event.preventDefault();
+
+		// Saving number of Touch start happends (number of fingers on teh button,
+	  // though most of the time will be 1, don't won't a nasty bug like tooltip stuck open)
+		// Starts by 1 and increases each time this func is called
+		elem._numOfTouches = event.touches.length;
+
+		// On first tap start counting
+		if (elem._numOfTouches === 1) {
+			// Timeout will be canceled by `_handleTouchEndFor` if a short tap have occurred
+			elem._timeout = setTimeout(function () {
+				elem._toolTipIsOpen = true;
+				BattleTooltips.showTooltipFor(roomid, thing, type, elem, ownHeight);
+			}, touchDelayForTap);
+		}
+	};
+	BattleTooltips._handleTouchLeaveFor = function (event, elem) {
+		// Prevent default on touch events to block mouse events when touch is used
+		event.preventDefault();
+
+		elem._numOfTouches = event.touches.length;
+		// Delete variable when last finger left, no need for memory
+		if (elem._numOfTouches === 0) {
+			delete elem._numOfTouches;
+		}
+
+		// If tooltip is open and the last finger left, close the tooptip
+		if (elem._toolTipIsOpen && elem._numOfTouches === undefined) {
```

Comment:
I wouldn't consider `._toolTipIsOpen` the best way to track an open tooltip. Since there should be only one open tooltip at a time, we should just track the open tooltip globally.